static_src/util/health.js

```diff
@@ -0,0 +1,63 @@
+
+import { appStates, entityHealth } from '../constants';
+
+const HEALTHY_STATES = [
+  entityHealth.inactive,
+  entityHealth.ok
+];
+
+const HEALTH_RANKED = [
+  entityHealth.error,
+  entityHealth.warning,
+  entityHealth.ok,
+  entityHealth.inactive
+];
+
+
+export function appHealth(app) {
+  if (!app) {
+    throw new Error('`app` must be provided.');
+  }
+
+  if (app.state === appStates.started && app.running_instances === app.instances) {
+    return entityHealth.ok;
+  }
+
+  if (app.running_instances === 0) {
```

Comment:
Are we sure theres no case where running instances would be more then 1 and app state would be stopped?